### DISCUSSION TITLE
Starting fireflyiii_app_1 ... error

### DISCUSSION THREAD

**Discussion by roberthchan** [firefly-iii/firefly-iii#5430] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Hello all.
I'm using a ionos vps (ionos image docker):

CentOS Linux 8 x86_64
Host: VMware Virtual Platform None
Kernel: 4.18.0-348.2.1.el8_5.x86_64
Uptime: 5 hours, 13 mins
Packages: 715 (rpm)
Shell: bash 4.4.20
Resolution: preferred
Terminal: /dev/pts/0
CPU: Intel Xeon Gold 5120 (1) @ 2.194GHz
GPU: 00:0f.0 VMware SVGA II Adapter
Memory: 259MiB / 438MiB

When i follow the intruccions in:
https://docs.firefly-iii.org/firefly-iii/installation/docker/
Docker Compose: downloading and using mentioned files, send me a error.

Before that i install Docker compose from:
https://docs.docker.com/compose/install/

### Expected behaviour

Firefly running

### Debug information

[root@localhost fireflyIII]# docker-compose up -d
fireflyiii_db_1 is up-to-date
Starting fireflyiii_app_1 ... error

ERROR: for fireflyiii_app_1  Cannot start service app: failed to create endpoint fireflyiii_app_1 on network fireflyiii_default: failed to add the host (vethad1d475) <=> sandbox (veth6d2f88f) pair interfaces: cannot allocate memory

ERROR: for app  Cannot start service app: failed to create endpoint fireflyiii_app_1 on network fireflyiii_default: failed to add the host (vethad1d475) <=> sandbox (veth6d2f88f) pair interfaces: cannot allocate memory
ERROR: Encountered errors while bringing up the project.

### Steps to reproduce

docker-compose up -d

### Additional info

[filesComposer.zip](https://github.com/firefly-iii/firefly-iii/files/7776641/filesComposer.zip)

**Comment by JC5** [comment]
> pair interfaces: cannot allocate memory

That's the problem. That's not on me I'm afraid.
